src/main/java/frc/robot/RobotContainer.java

```diff
@@ -27,7 +27,6 @@
 import frc.robot.swerve.SwerveDrive;
 
 public class RobotContainer {
-
```

Comment:
?

Reply:
This has been fixed when @vichik123 merged develop to commandGroups
f5a59e4

Reply:
Thank you for providing context on many of the comments! It helps me to understand stuff better. Note however, that the commit you referenced doesn't include changes to this file, and this diff still exists in this branch.

Reply:
![image](https://github.com/Galaxia5987/robot-2024/assets/68773850/995f8e86-2403-4f32-95ef-08a812d06f26)

Reply:
Sorry, this is something spotless did